packages/config/config/devices/0x0115/zme_06433.json

```diff
@@ -138,10 +138,9 @@
 		"5": {
 			"label": "Dimming speed",
 			"description": "Time to dim on button presses",
-			"unit": "10ms",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 30,
```

Comment:
no

Reply:
Get rid of the options instead

Reply:
These maxes come from the file. Are you sure it can handle 255?

Reply:
I don't see any config parameters in the file https://products.z-wavealliance.org/products/19/json
Are you sure you didn't infer that from the option values?